main.tf

```diff
@@ -77,7 +58,7 @@ resource "azurerm_application_gateway" "ag" {
     for_each = [for app in local.gateways[count.index].app_configuration : {
       name          = "${app.product}-${app.component}"
       path          = lookup(app, "health_path_override", "/health/liveness")
-      host_name     = join(".", [lookup(app, "host_name_prefix", "${app.product}-${app.component}-${var.env}"), local.gateways[count.index].gateway_configuration.host_name_suffix])
+      host_name     = join(".", [lookup(app, "host_name_prefix", "${app.product}-${app.component}"), local.gateways[count.index].gateway_configuration.host_name_suffix])
```

Comment:
you can't just remove this, it changes the hostname and would break every CFT app if this was merged, also see: https://github.com/hmcts/terraform-module-application-backend/pull/26

this probably needs a configuration flag, `include_env_in_app_name` maybe? thoughts @adusumillipraveen ?

Reply:
yeah, I guess we cannot avoid , we at least need this at module level so that each entry don't need to repeat them ?

Reply:
yeah i think so